doc/faq/installing_faq.rst

```diff
@@ -267,14 +178,14 @@ retry the install.  If that does not work, then check
 Installing via OSX mpkg installer package
 -----------------------------------------
 
-matplotlib also has a disk image (``.dmg``) installer, which contains a
-typical Installer.app package to install matplotlib.  You should use binary
+Matplotlib also has a disk image (``.dmg``) installer, which contains a
```

Comment:
This is no longer true, we only do wheels on mac now.  Can you remove all references to `dmg` files?

Reply:
Can you just edit accordingly?  I have close to no idea how things work on OSX, so I can cargo it but I'd rather not.

Reply:
I _also_ have no idea how macs work, but I do know we do not build a dmg file.

Reply:
just killed the section